packages/logging/src/internal/log-tags.ts

```diff
@@ -19,7 +19,7 @@ import safeStringify from 'fast-safe-stringify'
 import { byName as colorsByName } from './colors'
 
 export type PrimitiveType = string | number | boolean
-type LogTagValue = PrimitiveType | undefined | Record<string, unknown>
+type LogTagValue = PrimitiveType | undefined | Record<string, unknown> | string[]
```

Comment:
I think this is better. I also think we should add a test for it in pino.test.ts in the logTag section, we need to verify how this is serialized in the json format and in the text format. I _think_ this might require a bit of work in the text format.
```suggestion
type LogTagValue = PrimitiveType | undefined | Record<string, unknown> | Array<PrimitiveType>
```